### PR TITLE
Issue-79 Added contextual targeting support

### PR DESCRIPTION
**Pull request by gldiazcardenas**
https://github.com/taboola/backstage-api-java-client/issues/79

Endpoints:

- Read Contextual Segments
- Read Contextual Segments by Country
- Update Contextual Segment Targeting
- Read Contextual Segment Targeting

### CODE REVIEW

File: src/main/java/com/taboola/backstage/model/dictionary/ContextualSegment.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2023 InPowered. All Rights Reserved.
```
Comment (vladimanaev) [inline src/main/java/com/taboola/backstage/model/dictionary/ContextualSegment.java:2]: Please remove Copyright header

File: src/main/java/com/taboola/backstage/model/dictionary/ContextualSegment.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+ *
+ * This software is the proprietary information of InPowered.
+ * Use is subject to license terms.
+ */
+package com.taboola.backstage.model.dictionary;
+
+/**
+ * @author Gabriel Diaz, 21/11/23.
+ */
+public class ContextualSegment {
```
Comment (vladimanaev) [inline src/main/java/com/taboola/backstage/model/dictionary/ContextualSegment.java:12]: Please impl toString method

**Review by vladimanaev — CHANGES_REQUESTED** [review]
- Missing toString method on class ContextualSegment
- Copyright header should be removed from class ContextualSegment

Everything else seem good.

**Comment by gldiazcardenas** [comment]
Changes done

**Review by vladimanaev — APPROVED** [review]

**Pull request merged**